indy-besu/docs/design/did-doc.md

```diff
@@ -0,0 +1,261 @@
+# DID Method
+
+## DID Syntax
+| parameter          | value                                                   |
+|--------------------|---------------------------------------------------------|
+| did                | “did:” method-name “:” namespace “:” method-specific-id |
+| method-name        | "indy2"                                                 |
+| namespace          | “testnet”/"mainnet"                                     |
+| method-specific-id | indy-id / UUID                                          |
+| indy-id            | Base58(Truncate_msb(16(SHA256(publicKey))))             |
```

Comment:
1. Public key here means the first verification method added to DID Document. It allows making DID Document self-trusted. (Ed25519 signature validation is non-trivial and planned for the next phases)
https://hyperledger.github.io/indy-did-method/#nym-transaction-version Indy DID method about identifiers:
> 1: Validation is performed according to the did:sov method, in which the DID must be the first 16 bytes of the Verification Method public key.
> 2: Validation is performed according to the did:indy, in which the namespace identifier component of the DID (last element) is derived from the initial public key of the DID, using the base58 encoding of the first 16 bytes of the SHA256 of the Verification Method public key (did = Base58(Truncate_msb(16(SHA256(publicKey))))). This DID is considered self-certifying.
2.  We added description for using DID Ethereum method contracts with Indy2 method. https://github.com/hyperledger/indy-node/blob/dca3cdcca4562f3bad38b107de5ab866361cab11/indy-besu/docs/design/did-methods.md
It will be possible to use permissioned modules for both methods. 
We didn't add Ethereum accounts as Indy2 identifiers not to bring a mess of differences with did:ethr.